client/src/assets/scss/foundation/_settings.scss

```diff
@@ -69,15 +69,15 @@ $global-font-size: 100%;
 $global-width: rem-calc(1200);
 $global-lineheight: 1.5;
 $foundation-palette: (
-  primary: #1779ba,
-  secondary: #767676,
+  primary: #41644A,
+  secondary: #263A29,
```

Comment:
we should add our custom styles to the `main.scss` file to keep track of Foundation styles vs ones we have overwritten